src/Authenticator/FormAuthenticator.php

```diff
@@ -80,17 +80,10 @@ protected function _getData(ServerRequestInterface $request)
      */
     protected function _buildLoginUrlErrorResult($request)
     {
-        $config = $this->getConfig('urlChecker');
-        if (isset($config['checkFullUrl']) && $config['checkFullUrl']) {
-            $url = (string)$request->getUri();
-        } else {
-            $url = $request->getUri()->getPath();
-        }
-
         $errors = [
             sprintf(
                 'Login URL `%s` did not match `%s`.',
-                $url,
+                (string)$request->getUri(),
```

Comment:
I thought about doing exactly the same, but then the test failed, you just changed the tests. I'm not sure if this is really what we want?

Reply:
We can't rely on fact that *every* checker should have full URL check option.